spec/board_spec.rb

```diff
@@ -33,12 +33,31 @@
   end
 
   describe '#valid_placement?' do
-    it 'can have a valid placement' do
+    it 'the ship length is the same at the number of coordinates' do
       cruiser = Ship.new("Cruiser", 3)
       submarine = Ship.new("Submarine", 2)
        expect(@board.valid_placement?(cruiser, ["A1", "A2"])).to be false
        expect(@board.valid_placement?(submarine, ["A2", "A3", "A4"])).to be false
-       expect(@board.valid_placement?(cruiser, ["A1", "A2", "A4"])).to be false
+       expect(@board.valid_placement?(cruiser, ["A1", "A2", "A3"])).to be true
+       expect(@board.valid_placement?(submarine, ["A2", "A3"])).to be true
+    end
+
+    it 'makes sure the coordinates are consecutive' do
```

Comment:
Coordinates are consecutive. Our test/ examples indicates both horizontal and vertical consecutives. Should we have a test samples for both horizontal and vertical consecutives?